package.json

```diff
@@ -134,8 +134,7 @@
     "test": "vitest --mode=development",
     "test:rust": "(cd rust && just test && just lint)",
     "test:snapshots": "cross-env TARGET=web NODE_ENV=development playwright test --config=playwright.config.ts --grep=@snapshot --trace=on",
-    "test:unit": "vitest run --mode=development --exclude **/jest-component-unit-tests/*",
```

Comment:
Again, deleted the jest runtime stuff since vitest automatically supports the component testing as is.

Reply:
Yeah I should have done this in a separate PR. I wrote some component tests with vitest using this new structure.